apps/web/src/app/vault/components/collection-dialog/collection-dialog.component.html

```diff
@@ -64,6 +64,12 @@
           </bit-form-field>
         </bit-tab>
         <bit-tab label="{{ 'access' | i18n }}">
+          <div class="tw-mb-3">
+            {{ "grantCollectionAccess" | i18n }}
+            <span *ngIf="organization.allowAdminAccessToAllCollectionItems">{{
```

Comment:
This defaults to true for all current orgs, which means it'll show in production today if we merge this now. It should also depend on the v1 flag, e.g.

```suggestion
            <span *ngIf="(flexibleCollectionsV1Enabled$ | async) && organization.allowAdminAccessToAllCollectionItems">{{
```